config/checkstyle/checkstyle.xml

```diff
@@ -302,10 +302,5 @@
         <module name="CommentsIndentation">
             <property name="tokens" value="SINGLE_LINE_COMMENT, BLOCK_COMMENT_BEGIN"/>
         </module>
-        <module name="IllegalMethodCall">
```

Comment:
It looks like we can get rid of `checkstyle("com.thomasjensen.checkstyle.addons:checkstyle-addons:7.0.1")`